src/igvfd/schemas/file.json

```diff
@@ -230,6 +230,16 @@
         "accession": {
             "accessionType": "FI"
         },
+        "analysis_step_version": {
+            "title": "Analysis step version",
```

Comment:
Title should be capitalized Analysis Step Version

Reply:
done